BeeSwift/TimerViewController.swift

```diff
@@ -9,18 +9,26 @@
 import UIKit
 import SnapKit
 import MBProgressHUD
+import OSLog
 
 import BeeKit
 
 class TimerViewController: UIViewController {
+    let logger = Logger(subsystem: "com.beeminder.beeminder", category: "TimerViewController")
+
     let timerLabel = BSLabel()
     let startStopButton = BSButton(type: .system)
     let goal: Goal
     var timingSince: Date?
     var timer: Timer?
     var units: String?
     var accumulatedSeconds = 0
-
+    
+    private var idleTimer: Timer?
+    private var defaultBrightnessLevel: CGFloat = UIScreen.main.brightness
+    private let dimmedBrightnessLevel: CGFloat = 0.2
```

Comment:
Perhaps this should be a function of the default brightness level? If the user has brightness turned low this could end up increasing it, right?